### ISSUE TITLE
Comments/thoughts on the documentation

### ISSUE THREAD

**Issue by jeveki** [reproducible-reporting/stepup-reprep#1]
I am dumping here some thoughts on the documentation as I go through them (more are likely to come). Some are trivial, all are up for discussion obviously, feel free to dismiss any. It may be useful to make a new issue for some major ones, but I think it makes more sense to vet first instead of opening multiple minor issues. I certainly want to contribute to some of the more important ones if they are deemed relevant.

- [ ] I find the organisation of installation instructions somewhat confusing. There is now an installation tab that is very minimal (pip install stepup-reprep). In practice, I think you will actually almost always need the steps under 'Tutorials > Before you begin' before you can do anything useful. This tab is not the first place I would look to get to a working installation. Furthermore, the 'before you begin' tab does not include the actual installation instruction of stepup which you still need. I wonder if it would make sense to move the 'before you begin' tab to the installation tab, underneath the part that is already there. (See #9)
- [ ] I did not see the point of the screencast tutorials until I realised there were some comments underneath (I was watching full screen), would be useful to make subtitles on screen instead of comments underneath the video. (See #7)
- [ ] #8
- [ ] The documentation now says very 'sec' to avoid Jupyter Notebooks. While I understand the point in this context, I think that many potential users heavily rely on Jupyter Notebooks for their (early) analyses. This may deter them from using this. I would rephrase the statement to make it softer and explain the reasoning. (See #6)
- [x] The documentation is currently geared towards computational research, which we may want to broaden. Example [here](https://reproducible-reporting.github.io/stepup-reprep/tutorials/template_conventions/#:~:text=Expensive%20calculations%20done%20previously.). (See 4399cc2)
- [ ] The documentation sounds sometimes quite directive, which is good for internal use (what you expect of your students and collaborators), but which external people might find does not represent their use case. Example [here](https://reproducible-reporting.github.io/stepup-reprep/tutorials/template_conventions/#:~:text=Data%20generated%20with%20closed%2Dsource%20software.%20(Avoid%20closed%2Dsource%20software%20when%20you%20have%20the%20choice.)). Many experimentalists are forced to use closed software related to their equipment. (See #6)
- [ ] Maybe [this statement](https://reproducible-reporting.github.io/stepup-reprep/tutorials/template_conventions/#:~:text=HDF5%20files%20due%20to%20their%20data%20integrity%20issues.) (on data integrity issues of HDF5 files) could use a link? (See #2)

**Comment by tovrstra** [comment]
Thanks for all the helpful comments so far! Indeed, some of the points definitely deserve their own issue, but all of them should be addressed. I'll create new issues for those points that could be some benefit from having a discussion.

**Comment by tovrstra** [comment]
I've converted all points to issues and tried to address one. I'm closing this one.
